setup.cfg

```diff
@@ -33,6 +36,7 @@ nlp =
 test =
   pytest==7.1.3
   moto[s3]==4.0.7
+  shapely==2.0.2
```

Comment:
```suggestion
```
this is already included by %(s3)s as it is added in line 24

Reply:
Woops missed that! Thanks!